src/qtiCommonRenderer/renderers/Math.js

```diff
@@ -38,6 +39,12 @@ export default {
     template: tpl,
     getContainer: containerHelper.get,
     render: function render(math) {
+        $('body').on('mathjaxRendered', function (event, reference) {
+            if ($(reference).find('math').length !== 0) {
+                $(reference).closest('.qti-choice').addClass('flexible-choice-width');
+            }
+        });
```

Comment:
**thought:** Unfortunately, I didn't review the original PR. This kind of specific code should not be added here. Otherwise, why bother adding this generic event? This code should have been placed inside the right consumer.

Anyway, it is too late now.